fxa/oauth.py

```diff
@@ -267,8 +267,12 @@ def verify_token(self, token, scope=None):
             # change.
             # https://github.com/mozilla/PyFxA/issues/81 is an issue about
             # getting the jwks url out of the openid-configuration.
-
-            keys = self.apiclient.get('/jwks').get('keys', [])
+            keys = []
+            pyfxa_jwks = os.getenv("PYFXA_JWKS", None)
+            if pyfxa_jwks is not None:
+                keys.append(json.loads(pyfxa_jwks))
```

Comment:
I think we're going to want `PYFXA_JWKS` to be a list of keys (otherwise it's going to be hard to manage a future key rotation without downtime) so we should make this a `keys.extend`.

You could also consider doing this json parsing once, in `__init__`, and then accessing this via `self._jwks` or similar. But I don't fell strongly about it.